app/graphql/schema.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import strawberry
+from astropy.time import Time
+from typing import List
+from datetime import datetime
+
+from app.process_manager import TaskType
+from app.scheduler import Scheduler
+from app.common.minimodel import Site
+from app.process_manager import ProcessManager as PM
+from app.plan_manager import PlanManager
+from app.config import config
+from .scalars import CreateNewScheduleInput, SPlans
+
+
+# TODO: All times need to be in UTC. This is done here but converted from the Optimizer plans, where it should be done.
+
+
+@strawberry.type
+class Mutation:
+    @strawberry.mutation
+    async def new_schedule(self,
+                           new_schedule_input: CreateNewScheduleInput) -> bool:
+        try:
+            start, end = Time(new_schedule_input.start_time, format='iso', scale='utc'), Time(new_schedule_input.end_time, format='iso', scale='utc')
+        except ValueError:
+            raise ValueError("Invalid time format. Must be ISO8601.")
+        manager = PM()
```

Comment:
Why did we change the name to `PM`? It seems really unintuitive to me.

Reply:
Copilot suggestion. It's changed now.

Reply:
👍